specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/examples/Patches_Get.json

```diff
@@ -0,0 +1,30 @@
+{
+  "operationId": "Patches_Get",
+  "title": "Patches_Get_0",
+  "parameters": {
+    "api-version": "2023-05-02-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "rg",
+    "builderName": "testBuilder",
+    "patchName": "testPatch"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "patchId": "1ce4b2a0",
+            "targetImageName": "testregistry.azurecr.io/test-image:latest",
+            "newRunImage": "mcr.microsoft.com/oryx/builder:run-dotnet-aspnet-7.0.7-cbl-mariner2.0",
+            "oldRunImage": "mcr.microsoft.com/oryx/builder:run-dotnet-aspnet-7.0.5-cbl-mariner2.0",
+            "reason": "Newer run image available.",
+            "targetResourceGroup": "test-rg",
```

Comment:
nit: the `targetResourceGroup` value here is different than the one provided as a part of the parameters to the request -- is that intentional?

Reply:
ah sorry this one is a typo. will be fixed in the next commit.